### ISSUE TITLE
Audit telemetry ping vs docs

### ISSUE THREAD

**Issue by pdehaan** [mozilla/testpilot#940]
According to the [docs/README-METRICS.md](https://github.com/mozilla/testpilot/blob/master/docs/README-METRICS.md#data-collection-details) docs, our Telemetry ping should look something like this:

``` js
{
  "agent": "User Agent String",
  "version": 1,  // Just in case we need to drastically change the format later
  "tests": {
    "universal_search": { // The em:id field from the add-on
      "last_enabled": 1457462200,
      "last_disabled": 1457461100,
      "features": {}
    },
    "page_shot": {
      "last_enabled": 1457462200,
      "last_disabled": 1457461100,
      "features": {}
    }
  }
}
```

I went through and took a look at some of the Telemetry pings that I've been sending, and they look _slightly_ different, but I'm not sure if the docs aren't 100% accurate or if we're not sending the right fields in the Telemetry ping payload.

``` json
{
  "version": 1,
  "tests": {
    "@activity-streams": {
      "features": {},
      "last_enabled": 1464903551507
    }
  }
}
```

Basically, I'm not seeing the [user] `agent` in our ping, and I'm not sure if we're sending the `last_disabled` data (or if it just isn't in any of my ping data because I haven't toggled any experiments).

<img width="850" alt="telemetry_data" src="https://cloud.githubusercontent.com/assets/557895/15764802/ecc6d42a-28e2-11e6-82b8-9c423c91b700.png">

**Comment by ghost** [comment]
We should also include how to use our testpilottest pings

**Comment by ghost** [comment]
Looks like this got fixed a while ago.  I verified my current pings line up with what is in the docs.

![tp](https://cloud.githubusercontent.com/assets/16946141/25872859/13b793fe-34c1-11e7-9c23-1bb0c1eb5de0.png)
![tpt](https://cloud.githubusercontent.com/assets/16946141/25872860/13bde9f2-34c1-11e7-866e-61984bf0ec18.png)
